frontend/src/pages/Sessions/SessionsFeedV3/SessionsFeedV3.tsx

```diff
@@ -195,6 +195,7 @@ export const SessionFeedV3 = React.memo(() => {
 				sessions: response.sessions_opensearch.sessions.map((s) => ({
 					...s,
 					payload_updated_at: new Date().toISOString(),
+					excluded: false,
```

Comment:
Not sure why this cropped up in this PR, but I was seeing some [build errors](https://github.com/highlight/highlight/actions/runs/4865025004/jobs/8674799124) from this property not being present.

Reply:
Looks like I can clean this up now after https://github.com/highlight/highlight/pull/5206. Will do that when I handle the merge conflicts.